cluster/gce/gci/configure-helper.sh

```diff
@@ -2399,6 +2399,22 @@ EOF
   fi
   if [[ "${ENABLE_METRICS_SERVER:-}" == "true" ]]; then
     setup-addon-manifests "addons" "metrics-server"
+    base_metrics_server_cpu="40m"
+    base_metrics_server_memory="40Mi"
+    metrics_server_memory_per_node="4"
+    metrics_server_min_cluster_size="16"
+    if [[ "${ENABLE_SYSTEM_ADDON_RESOURCE_OPTIMIZATIONS:-}" == "true" ]]; then
```

Comment:
Is there a reason why you want to put a no-op logic here?

Reply:
Are you saying that you want to check in unnecessary code to make local testing easier, or am I understanding this wrong?

Reply:
This code is necessary to support the final implementation of the functionality being added. It is not critical to this specific PR. My goal with this PR was to have the foundation ready for a follow-up PR that would update the numbers to be used when the ENABLE_SYSTEM_ADDON_RESOURCE_OPTIMIZATIONS environment variable is set to true. Given that I am now close to getting those numbers finalized I'll probably update this PR with those values as soon as I've got approval from the appropriate stakeholders. In the meantime, I'd still like to have the rest of the changes reviewed in advance. Thanks.